tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1012,6 +1014,7 @@ def test_restore_object_log_to_db_fails_does_not_halt(
         restore_expire_days = randint(0, 99)  # nosec
         retrieval_type = uuid.uuid4().__str__()
         mock_s3_cli = Mock()
+        mock_s3_cli.restore_object.return_value = {"ResponseMetadata": {"HTTPStatusCode": 200}}
```

Comment:
The status value should be 202 on an initial restore. 200 occurs if the granule has already been restored.

Reply:
Fixed